quinn/src/streams.rs

```diff
@@ -347,6 +347,11 @@ where
         }
     }
 
+    /// [`read_chunk()`]: RecvStream::read_chunk
```

Comment:
This should be documented, including an explicit warning against relying on the chunk boundaries having any particular relationship to the sender's writes.

Reply:
I got this a better comment. I'm not sure I grasped everything you'd like it to say?